app/models/reminder_email_mentor.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class ReminderEmailMentor < TrackedEmail
+  def mail_to_send
+    UserMailer.mentor_sign_in_reminder_email(user)
+  end
+
+  def send!(force_send: false)
```

Comment:
If only a dev ever uses this, I am not sure we need the `force_send`. But if we do need it, unit tests please :)

Reply:
I left this is thinking whether all the mentors be registered? I don't think they will. We can remove the `registered_participant` check, again left this in for potential future email sends. Don't mind either way but will add force_send tests